modules/supported-platforms-for-managed-clusters.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: PROCEDURE
+[id="supported-platforms-for-managed-clusters_{context}"]
+= supported-platforms-for-managed-clusters
+
+.Prerequisites
+
+The Assisted Installer currently supports the following OpenShift platforms:
+- BareMetal
+- VSphere
+- None
+
+.Procedure
+
+. Select the platform in the AgentClusterInstall CR, via `spec.platformType`.
+
+In some cases, the system will automatically set the platform type:
+- The default platform for multi-node clusters is BareMetal
+- The default platform for Single Node OpenShift is None
+- Enabling `spec.networking.userManagedNetworking` without specifying the platform will cause the platform to be None.
+BareMetal platform and userManagedNetworking are not compatible.
+
+[NOTE]
+====
+vSphere is not supported for single node OpenShift. And similar to BareMetal, it uses UPI installation in regards to VMs provisioning should be done by the user. Also, VIPs DHCP allocation should be configured manually in vSphere env, maybe just a gritty detail:)
```

Comment:
@slaviered @danielerez can you verify if the sentence regarding UPI installation is correct? I'm not sure

Reply:
Yeah, for vSphere the uses should provide the VMs for the cluster. The Assisted Installer configures the network and verifies the infrastructure before the installation to ensure a successful installation.  VIPs DHCP allocation should indeed be manually configured in vSphere.